charts/airflow/files/pod_template.kubernetes-helm-yaml

```diff
@@ -74,6 +79,9 @@ spec:
       volumeMounts:
         {{- $volumeMounts | indent 8 }}
       {{- end }}
+    {{- if .Values.airflow.kubernetesPodTemplate.extraContainers }}
+      {{- toYaml .Values.airflow.kubernetesPodTemplate.extraContainers | nindent 4 }}
+    {{- end }}
```

Comment:
We already have a PR that adds this feature (https://github.com/airflow-helm/charts/pull/456), so we might want to break up your PR and just have it add the `airflow.kubernetesPodTemplate.shareProcessNamespace` value.

Reply:
As https://github.com/airflow-helm/charts/pull/456 will add this feature, please remove these changes.